### PR TITLE
home_15, home_16, home_17

### PR DESCRIPTION
**Pull request by sanyalyao**
Please check out home_15 and home_16

### CODE REVIEW

File: home_15/Pages/AccountPage.cs [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+        private Button deleteAccountButton = new Button(By.CssSelector("runtime_platform_actions-action-renderer[apiname='Delete'] *> span"));
+        private Button confirmDeleteAccountButton = new Button("button", "title", "Delete");
+
+        private Input accountNameInput = new Input("input", "name", "Name");
+        private Input phoneInput = new Input("input", "name", "Phone");
+        private Input accountNumberInput = new Input("input", "name", "AccountNumber");
+        private Input billingStreetInput = new Input("textarea", "name", "street");
+        private Input billingZipInput = new Input("input", "name", "postalCode");
+        private Input billingCityInput = new Input("input", "name", "city");
+        private Input billingCountryInput = new Input("input", "name", "country");
```
Comment (VladimirVolkovD) [inline home_15/Pages/AccountPage.cs:39]: is  it account page?

File: home_15/Pages/ContactPage.cs [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+        private Button confirmDeleteContactButton = new Button("button", "title", "Delete");
+
+        private Input firstNameInput = new Input("input", "name", "firstName");
+        private Input lastNameInput = new Input("input", "name", "lastName");
+        private Input mobileInput = new Input("input", "name", "MobilePhone");
+        private Input emailInput = new Input("input", "name", "Email");
+        private Input mailingStreetInput = new Input("textarea", "name", "street");
+        private Input mailingZipInput = new Input("input", "name", "postalCode");
+        private Input mailingCityInput = new Input("input", "name", "city");
+        private Input mailingCountryInput = new Input("input", "name", "country");
```
Comment (VladimirVolkovD) [inline home_15/Pages/ContactPage.cs:40]: same

**Review by VladimirVolkovD — COMMENTED** [review]
main logic basen on try catch approach, not the best option

**Comment by sanyalyao** [comment]
Please check out fixes for home_15 and home_16 and added home_17

**Review by VladimirVolkovD — APPROVED** [review]

**Pull request merged**